{"code":13,"codeName":"Unauthorized","errorLabelSet":{},"errorResponse":{"code":13,"codeName":"Unauthorized","errmsg":"auth required, correlationID = 182960ca182f5c41c4a42094","ok":0},"level":"error","message":"Erro ao buscar histórias: auth required, correlationID = 182960ca182f5c41c4a42094","ok":0,"stack":"MongoServerError: auth required, correlationID = 182960ca182f5c41c4a42094\n    at Connection.sendCommand (/home/josecicero/Documentos/Project/my-history/node_modules/mongodb/src/cmap/connection.ts:550:17)\n    at processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async Connection.command (/home/josecicero/Documentos/Project/my-history/node_modules/mongodb/src/cmap/connection.ts:624:22)\n    at async Server.command (/home/josecicero/Documentos/Project/my-history/node_modules/mongodb/src/sdam/server.ts:337:21)\n    at async FindOperation.execute (/home/josecicero/Documentos/Project/my-history/node_modules/mongodb/src/operations/find.ts:130:12)\n    at async tryOperation (/home/josecicero/Documentos/Project/my-history/node_modules/mongodb/src/operations/execute_operation.ts:283:14)\n    at async executeOperation (/home/josecicero/Documentos/Project/my-history/node_modules/mongodb/src/operations/execute_operation.ts:115:12)\n    at async FindCursor._initialize (/home/josecicero/Documentos/Project/my-history/node_modules/mongodb/src/cursor/find_cursor.ts:92:22)\n    at async FindCursor.cursorInit (/home/josecicero/Documentos/Project/my-history/node_modules/mongodb/src/cursor/abstract_cursor.ts:944:21)\n    at async FindCursor.fetchBatch (/home/josecicero/Documentos/Project/my-history/node_modules/mongodb/src/cursor/abstract_cursor.ts:982:7)","timestamp":"2025-03-03T19:12:32.229Z"}
{"level":"error","message":"Erro ao conectar ao MongoDB: mongodb+srv URI cannot have port number","stack":"MongoParseError: mongodb+srv URI cannot have port number\n    at new ConnectionString (/home/josecicero/Documentos/Project/my-history/node_modules/mongodb-connection-string-url/src/index.ts:201:15)\n    at parseOptions (/home/josecicero/Documentos/Project/my-history/node_modules/mongodb/src/connection_string.ts:261:15)\n    at new MongoClient (/home/josecicero/Documentos/Project/my-history/node_modules/mongodb/src/mongo_client.ts:387:32)\n    at NativeConnection.createClient (/home/josecicero/Documentos/Project/my-history/node_modules/mongoose/lib/drivers/node-mongodb-native/connection.js:328:14)\n    at NativeConnection.openUri (/home/josecicero/Documentos/Project/my-history/node_modules/mongoose/lib/connection.js:1071:34)\n    at Mongoose.connect (/home/josecicero/Documentos/Project/my-history/node_modules/mongoose/lib/mongoose.js:446:15)\n    at Object.<anonymous> (/home/josecicero/Documentos/Project/my-history/src/server.ts:20:4)\n    at Module._compile (node:internal/modules/cjs/loader:1364:14)\n    at Module.m._compile (/home/josecicero/Documentos/Project/my-history/node_modules/ts-node/src/index.ts:1618:23)\n    at Module._extensions..js (node:internal/modules/cjs/loader:1422:10)","timestamp":"2025-03-03T19:16:59.027Z"}
{"code":13,"codeName":"Unauthorized","errorLabelSet":{},"errorResponse":{"code":13,"codeName":"Unauthorized","errmsg":"auth required, correlationID = 18296184ca700902093393a8","ok":0},"level":"error","message":"Erro ao buscar histórias: auth required, correlationID = 18296184ca700902093393a8","ok":0,"stack":"MongoServerError: auth required, correlationID = 18296184ca700902093393a8\n    at Connection.sendCommand (/home/josecicero/Documentos/Project/my-history/node_modules/mongodb/src/cmap/connection.ts:550:17)\n    at processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async Connection.command (/home/josecicero/Documentos/Project/my-history/node_modules/mongodb/src/cmap/connection.ts:624:22)\n    at async Server.command (/home/josecicero/Documentos/Project/my-history/node_modules/mongodb/src/sdam/server.ts:337:21)\n    at async FindOperation.execute (/home/josecicero/Documentos/Project/my-history/node_modules/mongodb/src/operations/find.ts:130:12)\n    at async tryOperation (/home/josecicero/Documentos/Project/my-history/node_modules/mongodb/src/operations/execute_operation.ts:283:14)\n    at async executeOperation (/home/josecicero/Documentos/Project/my-history/node_modules/mongodb/src/operations/execute_operation.ts:115:12)\n    at async FindCursor._initialize (/home/josecicero/Documentos/Project/my-history/node_modules/mongodb/src/cursor/find_cursor.ts:92:22)\n    at async FindCursor.cursorInit (/home/josecicero/Documentos/Project/my-history/node_modules/mongodb/src/cursor/abstract_cursor.ts:944:21)\n    at async FindCursor.fetchBatch (/home/josecicero/Documentos/Project/my-history/node_modules/mongodb/src/cursor/abstract_cursor.ts:982:7)","timestamp":"2025-03-03T19:25:54.090Z"}
{"code":13,"codeName":"Unauthorized","errorLabelSet":{},"errorResponse":{"code":13,"codeName":"Unauthorized","errmsg":"auth required, correlationID = 182961858cac569a093393ae","ok":0},"level":"error","message":"Erro ao buscar histórias: auth required, correlationID = 182961858cac569a093393ae","ok":0,"stack":"MongoServerError: auth required, correlationID = 182961858cac569a093393ae\n    at Connection.sendCommand (/home/josecicero/Documentos/Project/my-history/node_modules/mongodb/src/cmap/connection.ts:550:17)\n    at processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async Connection.command (/home/josecicero/Documentos/Project/my-history/node_modules/mongodb/src/cmap/connection.ts:624:22)\n    at async Server.command (/home/josecicero/Documentos/Project/my-history/node_modules/mongodb/src/sdam/server.ts:337:21)\n    at async FindOperation.execute (/home/josecicero/Documentos/Project/my-history/node_modules/mongodb/src/operations/find.ts:130:12)\n    at async tryOperation (/home/josecicero/Documentos/Project/my-history/node_modules/mongodb/src/operations/execute_operation.ts:283:14)\n    at async executeOperation (/home/josecicero/Documentos/Project/my-history/node_modules/mongodb/src/operations/execute_operation.ts:115:12)\n    at async FindCursor._initialize (/home/josecicero/Documentos/Project/my-history/node_modules/mongodb/src/cursor/find_cursor.ts:92:22)\n    at async FindCursor.cursorInit (/home/josecicero/Documentos/Project/my-history/node_modules/mongodb/src/cursor/abstract_cursor.ts:944:21)\n    at async FindCursor.fetchBatch (/home/josecicero/Documentos/Project/my-history/node_modules/mongodb/src/cursor/abstract_cursor.ts:982:7)","timestamp":"2025-03-03T19:25:57.310Z"}
{"cause":{"code":"ERR_INVALID_URL","input":"iLoveJS://admin:98034183"},"errorLabelSet":{},"level":"error","message":"Erro ao conectar ao MongoDB: Unable to parse admin:98034183 with URL","stack":"MongoRuntimeError: Unable to parse admin:98034183 with URL\n    at new HostAddress (/home/josecicero/Documentos/Project/my-history/node_modules/mongodb/src/utils.ts:898:28)\n    at fromString (/home/josecicero/Documentos/Project/my-history/node_modules/mongodb/src/utils.ts:947:12)\n    at Array.map (<anonymous>)\n    at parseOptions (/home/josecicero/Documentos/Project/my-history/node_modules/mongodb/src/connection_string.ts:266:43)\n    at new MongoClient (/home/josecicero/Documentos/Project/my-history/node_modules/mongodb/src/mongo_client.ts:387:32)\n    at NativeConnection.createClient (/home/josecicero/Documentos/Project/my-history/node_modules/mongoose/lib/drivers/node-mongodb-native/connection.js:328:14)\n    at NativeConnection.openUri (/home/josecicero/Documentos/Project/my-history/node_modules/mongoose/lib/connection.js:1071:34)\n    at Mongoose.connect (/home/josecicero/Documentos/Project/my-history/node_modules/mongoose/lib/mongoose.js:446:15)\n    at Object.<anonymous> (/home/josecicero/Documentos/Project/my-history/src/server.ts:20:4)\n    at Module._compile (node:internal/modules/cjs/loader:1364:14)","timestamp":"2025-03-03T22:00:48.755Z"}
{"_message":"Historia validation failed","errors":{"conteudo":{"kind":"required","message":"Path `conteudo` is required.","name":"ValidatorError","path":"conteudo","properties":{"message":"Path `conteudo` is required.","path":"conteudo","type":"required","value":""},"value":""},"titulo":{"kind":"required","message":"Path `titulo` is required.","name":"ValidatorError","path":"titulo","properties":{"message":"Path `titulo` is required.","path":"titulo","type":"required","value":""},"value":""}},"level":"error","message":"Erro ao salvar a história: Historia validation failed: titulo: Path `titulo` is required., conteudo: Path `conteudo` is required.","stack":"ValidationError: Historia validation failed: titulo: Path `titulo` is required., conteudo: Path `conteudo` is required.\n    at model.Document.invalidate (/home/josecicero/Documentos/Project/my-history/node_modules/mongoose/lib/document.js:3329:32)\n    at /home/josecicero/Documentos/Project/my-history/node_modules/mongoose/lib/document.js:3090:17\n    at /home/josecicero/Documentos/Project/my-history/node_modules/mongoose/lib/schemaType.js:1407:9\n    at processTicksAndRejections (node:internal/process/task_queues:77:11)","timestamp":"2025-03-03T22:01:20.032Z"}
{"level":"error","message":"Erro ao buscar histórias: Operation `historias.find()` buffering timed out after 10000ms","stack":"MongooseError: Operation `historias.find()` buffering timed out after 10000ms\n    at Timeout.<anonymous> (/home/josecicero/Documentos/Project/my-history/node_modules/mongoose/lib/drivers/node-mongodb-native/collection.js:187:23)\n    at listOnTimeout (node:internal/timers:569:17)\n    at processTimers (node:internal/timers:512:7)","timestamp":"2025-03-03T22:01:20.964Z"}
{"cause":{"code":"ERR_INVALID_URL","input":"iLoveJS://admin:98034183"},"errorLabelSet":{},"level":"error","message":"Erro ao conectar ao MongoDB: Unable to parse admin:98034183 with URL","stack":"MongoRuntimeError: Unable to parse admin:98034183 with URL\n    at new HostAddress (/home/josecicero/Documentos/Project/my-history/node_modules/mongodb/src/utils.ts:898:28)\n    at fromString (/home/josecicero/Documentos/Project/my-history/node_modules/mongodb/src/utils.ts:947:12)\n    at Array.map (<anonymous>)\n    at parseOptions (/home/josecicero/Documentos/Project/my-history/node_modules/mongodb/src/connection_string.ts:266:43)\n    at new MongoClient (/home/josecicero/Documentos/Project/my-history/node_modules/mongodb/src/mongo_client.ts:387:32)\n    at NativeConnection.createClient (/home/josecicero/Documentos/Project/my-history/node_modules/mongoose/lib/drivers/node-mongodb-native/connection.js:328:14)\n    at NativeConnection.openUri (/home/josecicero/Documentos/Project/my-history/node_modules/mongoose/lib/connection.js:1071:34)\n    at Mongoose.connect (/home/josecicero/Documentos/Project/my-history/node_modules/mongoose/lib/mongoose.js:446:15)\n    at Object.<anonymous> (/home/josecicero/Documentos/Project/my-history/src/server.ts:20:4)\n    at Module._compile (node:internal/modules/cjs/loader:1364:14)","timestamp":"2025-03-03T22:01:53.236Z"}
{"code":59,"codeName":"CommandNotFound","errorLabelSet":{},"errorResponse":{"code":59,"codeName":"CommandNotFound","errmsg":"command insert not found, correlationID = 18296a2b08e9854f9a04d6ab","ok":0},"level":"error","message":"Erro ao salvar a história: command insert not found, correlationID = 18296a2b08e9854f9a04d6ab","ok":0,"stack":"MongoServerError: command insert not found, correlationID = 18296a2b08e9854f9a04d6ab\n    at Connection.sendCommand (/home/josecicero/Documentos/Project/my-history/node_modules/mongodb/src/cmap/connection.ts:550:17)\n    at processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async Connection.command (/home/josecicero/Documentos/Project/my-history/node_modules/mongodb/src/cmap/connection.ts:624:22)\n    at async Server.command (/home/josecicero/Documentos/Project/my-history/node_modules/mongodb/src/sdam/server.ts:337:21)\n    at async InsertOneOperation.executeCommand (/home/josecicero/Documentos/Project/my-history/node_modules/mongodb/src/operations/command.ts:179:12)\n    at async InsertOneOperation.execute (/home/josecicero/Documentos/Project/my-history/node_modules/mongodb/src/operations/insert.ts:54:12)\n    at async InsertOneOperation.execute (/home/josecicero/Documentos/Project/my-history/node_modules/mongodb/src/operations/insert.ts:84:17)\n    at async tryOperation (/home/josecicero/Documentos/Project/my-history/node_modules/mongodb/src/operations/execute_operation.ts:283:14)\n    at async executeOperation (/home/josecicero/Documentos/Project/my-history/node_modules/mongodb/src/operations/execute_operation.ts:115:12)\n    at async Collection.insertOne (/home/josecicero/Documentos/Project/my-history/node_modules/mongodb/src/collection.ts:285:12)","timestamp":"2025-03-03T22:04:24.060Z"}
{"code":59,"codeName":"CommandNotFound","errorLabelSet":{},"errorResponse":{"code":59,"codeName":"CommandNotFound","errmsg":"command insert not found, correlationID = 18296a8f8963ed2d09332b79","ok":0},"level":"error","message":"Erro ao salvar a história: command insert not found, correlationID = 18296a8f8963ed2d09332b79","ok":0,"stack":"MongoServerError: command insert not found, correlationID = 18296a8f8963ed2d09332b79\n    at Connection.sendCommand (/home/josecicero/Documentos/Project/my-history/node_modules/mongodb/src/cmap/connection.ts:550:17)\n    at processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async Connection.command (/home/josecicero/Documentos/Project/my-history/node_modules/mongodb/src/cmap/connection.ts:624:22)\n    at async Server.command (/home/josecicero/Documentos/Project/my-history/node_modules/mongodb/src/sdam/server.ts:337:21)\n    at async InsertOneOperation.executeCommand (/home/josecicero/Documentos/Project/my-history/node_modules/mongodb/src/operations/command.ts:179:12)\n    at async InsertOneOperation.execute (/home/josecicero/Documentos/Project/my-history/node_modules/mongodb/src/operations/insert.ts:54:12)\n    at async InsertOneOperation.execute (/home/josecicero/Documentos/Project/my-history/node_modules/mongodb/src/operations/insert.ts:84:17)\n    at async tryOperation (/home/josecicero/Documentos/Project/my-history/node_modules/mongodb/src/operations/execute_operation.ts:283:14)\n    at async executeOperation (/home/josecicero/Documentos/Project/my-history/node_modules/mongodb/src/operations/execute_operation.ts:115:12)\n    at async Collection.insertOne (/home/josecicero/Documentos/Project/my-history/node_modules/mongodb/src/collection.ts:285:12)","timestamp":"2025-03-03T22:11:35.717Z"}
{"code":59,"codeName":"CommandNotFound","errorLabelSet":{},"errorResponse":{"code":59,"codeName":"CommandNotFound","errmsg":"command insert not found, correlationID = 18296bcc268a4d5bc4a41335","ok":0},"level":"error","message":"Erro ao salvar a história: command insert not found, correlationID = 18296bcc268a4d5bc4a41335","ok":0,"stack":"MongoServerError: command insert not found, correlationID = 18296bcc268a4d5bc4a41335\n    at Connection.sendCommand (/home/josecicero/Documentos/Project/my-history/node_modules/mongodb/src/cmap/connection.ts:550:17)\n    at processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async Connection.command (/home/josecicero/Documentos/Project/my-history/node_modules/mongodb/src/cmap/connection.ts:624:22)\n    at async Server.command (/home/josecicero/Documentos/Project/my-history/node_modules/mongodb/src/sdam/server.ts:337:21)\n    at async InsertOneOperation.executeCommand (/home/josecicero/Documentos/Project/my-history/node_modules/mongodb/src/operations/command.ts:179:12)\n    at async InsertOneOperation.execute (/home/josecicero/Documentos/Project/my-history/node_modules/mongodb/src/operations/insert.ts:54:12)\n    at async InsertOneOperation.execute (/home/josecicero/Documentos/Project/my-history/node_modules/mongodb/src/operations/insert.ts:84:17)\n    at async tryOperation (/home/josecicero/Documentos/Project/my-history/node_modules/mongodb/src/operations/execute_operation.ts:283:14)\n    at async executeOperation (/home/josecicero/Documentos/Project/my-history/node_modules/mongodb/src/operations/execute_operation.ts:115:12)\n    at async Collection.insertOne (/home/josecicero/Documentos/Project/my-history/node_modules/mongodb/src/collection.ts:285:12)","timestamp":"2025-03-03T22:34:15.559Z"}
{"code":"ENOTFOUND","hostname":"_mongodb._tcp.atlas-cluster.mongodb.net","level":"error","message":"Erro ao conectar ao MongoDB: querySrv ENOTFOUND _mongodb._tcp.atlas-cluster.mongodb.net","stack":"Error: querySrv ENOTFOUND _mongodb._tcp.atlas-cluster.mongodb.net\n    at QueryReqWrap.onresolve [as oncomplete] (node:internal/dns/promises:275:17)","syscall":"querySrv","timestamp":"2025-03-03T22:40:17.173Z"}
